lib/DDG/Spice/Dota2.pm

```diff
@@ -0,0 +1,26 @@
+package DDG::Spice::Dota2;
+# ABSTRACT: Returns information about Dota 2 heroes and items.
+
+use DDG::Spice;
+
+name "Dota 2 Hero and Item Search";
```

Comment:
This metadata should be removed -- that all lives in https://duck.co/ia/view/dota2